dnf-behave-tests/features/module-update.feature

```diff
@@ -170,3 +170,114 @@ When I execute dnf with args "module update nodejs"
   And modules state is following
       | Module    | State     | Stream    | Profiles  |
       | nodejs    | enabled   | 11        |           |
+
+
+@not.with_os=rhel__eq__8
```

Comment:
All these tests are already passing on RHEL 8. Let's remove this tags and merge it.

Reply:
I removed the tags. Thanks.